### PR TITLE
PATCH Geometric Conversion

### PR DESCRIPTION
**Pull request by czaloom**

### CODE REVIEW

File: api/tests/functional-tests/backend/core/test_geometry.py [hunk 1]
```diff
@@ -27,6 +102,52 @@ def create_clf_dataset(db: Session, dataset_name: str):
     )
 
 
+@pytest.fixture
+def create_objdet_dataset(
```
Comment (ntlind) [inline api/tests/functional-tests/backend/core/test_geometry.py:106]: typo in function name

Comment (czaloom) [reply api/tests/functional-tests/backend/core/test_geometry.py:106]: short hand for object detection, ill write it full length

File: api/tests/functional-tests/backend/core/test_geometry.py [hunk 2]
```diff
@@ -92,3 +213,99 @@ def test_convert_geometry(
...
+        )
+    )
+    assert annotation_id is not None
+
+    q = _convert_raster_to_box([])
+    db.execute(q)
+
+    q = _convert_raster_to_polygon([])
+    db.execute(q)
+
```
Comment (ntlind) [inline api/tests/functional-tests/backend/core/test_geometry.py:250]: should we add a test for _convert_raster_to_multipolygon?

Comment (czaloom) [reply api/tests/functional-tests/backend/core/test_geometry.py:250]: Raster -> Multipolygon is not implemented as it is a non-trivial conversion. 

See #470 for reference.

**Review by ntlind — APPROVED** [review]

**Pull request merged**